src/PdfReader/useMeasure.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+
+declare const window: Window &
+  typeof globalThis & {
+    ResizeObserver: any;
+  };
+export type Dimensions = Pick<
+  DOMRectReadOnly,
+  'x' | 'y' | 'top' | 'left' | 'right' | 'bottom' | 'height' | 'width'
+>;
+export type UseMeasureRef<E extends Element = Element> = (element: E) => void;
+export type UseMeasureResult<E extends Element = Element> = [
```

Comment:
Seems like this is causing a lint error in the import of React - something to do with the react version?